packages/nimble-components/src/tree-item/index.ts

```diff
@@ -26,49 +25,11 @@ declare global {
  *
  */
 export class TreeItem extends FoundationTreeItem {
```

Comment:
While it isn't "necessary" I think it would still be good to have this and `AnchorTreeItem` implement `ISelectable`.

Reply:
Done